hail_search/__main__.py

```diff
@@ -7,12 +7,11 @@
 
 def run():
     logging.basicConfig(level=logging.INFO)
-    hl.init()
     app = init_web_app()
     web.run_app(
         app,
         host='0.0.0.0',  # nosec
-        port=5000,
+        port=8000,
```

Comment:
When running seqr locally, seqr is running on port 8000 , so this change makes it impossible to run seqr and search locally at the same time. Is there a reason for this change?

Reply:
I think I was actually looking for consistency _with_ the seqr app, because the service port itself doesn't matter if we're running a separate k8s service, but that doesn't make sense at all with local development.  I'll revert.